bmap/src/bmap/xml.rs

```diff
@@ -19,7 +19,7 @@ struct BlockMap {
 
 #[derive(Debug, Deserialize)]
 struct Bmap {
-    version: String,
+    //    version: String,
```

Comment:
it'd be nicer to ignore the lint errors here, not sure what the correct parameter is but `#[allow(dead_code)]` ?